intera_interface/src/intera_motion_interface/motion_controller_action_client.py

```diff
@@ -54,12 +54,16 @@ def send_trajectory(self, trajectory):
         goal.trajectory = trajectory
         self._client.send_goal(goal)
 
-    def wait_for_result(self):
+    def wait_for_result(self, timeout=None):
         """
         Wait for the current task to finish and then return the result
+        @param timeout: maximum time to block before returning
```

Comment:
What result is returned when the timeout is triggered? Please add to the comment. For example:
`@param timeout: maximum time to wait. If timeout is reached, return None.`